.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Docker images
+
+on:
+  pull_request:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [apache, fpm]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build ${{ matrix.image }} image
+        uses: docker/build-push-action@v2
+        with:
+          file: ${{ matrix.image }}/Dockerfile
```

Comment:
Hello! I would recommend building it for amd64 and arm64 to be able to run it eg. on M1 Macs as well
`platforms: linux/amd64,linux/arm64`

Reply:
I'd like to do this, but am hoping for this MR to be merged first so we can use smaller MRs to merge this kind of thing.